src/core/DataFile.cpp

```diff
@@ -1758,6 +1759,20 @@ void DataFile::upgrade_defaultTripleOscillatorHQ()
 	}
 }
 
+/*
+* This function aims to change the nodeNames of old project savings.
+*/
+void DataFile::upgrade_mixerRename()
+{
+	QDomNodeList fxmixer = elementsByTagName("fxmixer");
+
+	for(int i=0; !fxmixer.item(i).isNull(); ++i)
+	{
+		QDomNode tag = fxmixer.at(i);
+	    QDomElement mixer = tag.toElement();  
+	    mixer.setTagName("mixer");
+	}
```

Comment:
Fixing code formatting and simplifying, you can do like this:
```suggestion
	for (int i = 0; !fxmixer.item(i).isNull(); ++i)
	{
		fxmixer.item(i).toElement().setTagName("mixer");
	}
```